tests-e2e/util/WwwPage.ts

```diff
@@ -28,13 +28,16 @@ class WwwPage {
   /**
    * Return a Locator for a Course Card by its title.
    */
-  getCourseCard(title: string, { expectedCount = 1 } = {}): Locator {
+  async getCourseCard(
+    title: string,
+    { expectedCount = 1 } = {}
+  ): Promise<Locator> {
     const cardXPath = `div[${xPath.predicates.hasClass("course-card")}]`
-    const courseCard = this.page
+    const courseCard = await this.page
       .getByRole("link", { name: title })
       .locator(`${closest(cardXPath)} >> visible=true`)
```

Comment:
I'm fairly certain that this `await` does nothing: locators are evaluated lazily...there's nothing to really wait for until you use the locator for something.

The await below makes a lot of sense.